src/Lykke.Job.TransactionHandler/Handlers/OperationsCommandHandler.cs

```diff
@@ -76,16 +76,16 @@ public class OperationsCommandHandler
 
         public async Task<CommandHandlingResult> Handle(Commands.SaveTransferOperationStateCommand command, IEventPublisher eventPublisher)
         {
-            await _log.WriteInfoAsync(nameof(OperationsCommandHandler), nameof(Commands.SaveTransferOperationStateCommand), command.ToJson());
+            _log.WriteInfo(nameof(Commands.SaveTransferOperationStateCommand), command, "");
 
             var message = command.QueueMessage;
             var transactionId = message.Id;
 
             var transaction = await _transactionsRepository.FindByTransactionIdAsync(transactionId);
             if (transaction == null)
             {
-                await _log.WriteWarningAsync(nameof(OperationsCommandHandler), nameof(Commands.SaveManualOperationStateCommand), command.ToJson(), "unknown transaction");
-                return CommandHandlingResult.Fail(_retryTimeout);
+                _log.WriteError(nameof(Commands.SaveManualOperationStateCommand), command, new Exception($"unknown transaction {transactionId}"));
+                return CommandHandlingResult.Ok();
```

Comment:
Is this ok? Log a error, but return OK

Reply:
Yes, this is the point of this PR. In such case we have nothing to do, so retry is not needed — it's a pure error.